tests/benchmarks/multicore/idle_with_pwm/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _multicore_idle_with_pwm_test:
+
+PWM in low power states test
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This test benchmarks the idle behavior of an application that runs on multiple cores and uses the PWM driver to light up a LED.
+
+The test scenario repeats forever the following:
+
+* Gradually increase PWM duty cycle from 0% to 50% for one second (the LED lights up), power state active.
+* Set PWM duty cycle to 0% for one second (PWM disabled, the LED is OFF), power state low.
+
+Requirements
+************
+
+The test supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+Overview
+********
+
+The test demonstrates the use of PWM with low power modes.
+
+The code stored in the :file:`main.c` file is compiled for both application and radio cores.
+The application core uses pwm130 to generate PWM signal on **LED2**.
+The radio core uses pwm131 to generate PWM signal on GPIO port 0, pin 7.
+
+There are three test configurations in the :file:`testcase.yaml`.
+
+* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram'``
+   This configuration uses KConfigs that enable entering low power modes.
+   Logging is disabled.
+   Core sleeps for time sufficient to enter the ``suspend-to-ram`` power state.
+* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.sleep``
+   This configuration uses KConfigs that enable entering low power modes.
+   Logging is disabled.
+   Core sleeps for time sufficient to enter the ``suspend-to-idle``power state.
+* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.no_sleep``
+   You can use this configuration for debug purposes.
+   Logging is enabled while power mode is disabled.
```

Comment:
```suggestion
* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram``

  This configuration uses Kconfigs that enable entering low power modes.
  Logging is disabled.
  Core sleeps for time sufficient to enter the ``suspend-to-ram`` power state.
  
* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.sleep``

  This configuration uses Kconfigs that enable entering low power modes.
  Logging is disabled.
  Core sleeps for time sufficient to enter the ``suspend-to-idle``power state.
  
* ``benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.no_sleep``

  You can use this configuration for debug purposes.
  Logging is enabled while power mode is disabled.
```

Reply:
line 42 and 48 contain white character. Is this intentional?